lib/payday/invoiceable.rb

```diff
@@ -20,18 +20,22 @@ def bill_to
 
   # Calculates the subtotal of this invoice by adding up all of the line items
   def subtotal
-    line_items.reduce(BigDecimal.new("0")) { |result, item| result += item.amount }
+    line_items.reduce(BigDecimal.new("0")) { |result, item| result += item.amount_no_tax }
```

Comment:
@katiaAlexandrov going back to this branch - where is subtotal_with_items_tax and  items_tax going to be used? this should be changed on the backend as well?

Reply:
@alonhartal it's used internally, before the change it was only subtotal for line, so I've splitted It to amount_no_tax + item_tax + amount.
Total tax is sum of all lines item_tax
Subtotal is sum of all lines amount_no_tax.

Reply:
@katiaAlexandrov can you share a PDF example with the taxes?